pkg/apis/nodemaintenance/v1beta1/nodemaintenance_validator_test.go

```diff
@@ -3,6 +3,8 @@ package v1beta1
 import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	machinev1beta1 "github.com/openshift/machine-api-operator/pkg/apis/machine/v1beta1"
```

Comment:
nit picking: we usually only have core Go packages in 1st block of imports, can you move this is the next to next block please?